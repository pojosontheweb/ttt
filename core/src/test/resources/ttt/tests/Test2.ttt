<%@ page import="com.xyz.myapp.Foo" %>
<%!
    Foo foo;
    int bar;
%>
this is a <%= foo.getBar() %> template