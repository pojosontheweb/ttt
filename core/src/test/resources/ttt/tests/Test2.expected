package ttt.tests;

import com.xyz.myapp.Foo;

public class Test2 extends com.pojosontheweb.ttt.Template {

	private final Foo foo;
	private final int bar;

	public Test2(Foo foo, int bar) {
		this.foo = foo;
		this.bar = bar;
	}

	@Override
	public void render(java.io.Writer out) throws java.io.IOException {
		write(out, "this is a " );
		write(out,  foo.getBar()  );
		write(out, " template" );
	}
}
